processor/processor.go

```diff
@@ -332,7 +332,7 @@ func setupCountAs() {
 			// See if we can identify based on language name which is the most
 			// reliable as the name should be unique
 			for name := range languageDatabase {
-				if strings.ToLower(name) == strings.ToLower(t[1]) {
+				if strings.EqualFold(name, t[1]) {
```

Comment:
I fixed this one in #478. I can remove it from my PR. Not a problem

Reply:
I apologize for not noticing that this part duplicates your PR. Also thank you for your generosity in transferring these two lines of changes to me.

Reply:
I removed it from my PR

Reply:
Thank you :)